### PR TITLE
Tidy up scheduled maintenance functionality

### PR DESCRIPTION
**Pull request by alexmuller**
This fixes an old FIXME where we had a huge chunk of HTML stored in this repo. The new way to do this is to download a scheduled maintenance view from the static repo (like we do for our other error pages) and then use that instead.

### CODE REVIEW

File: modules/loadbalancer/manifests/maintenance.pp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Sets up a machine so that the loadbalancer can be put
+# into maintenance mode.
+#
+class loadbalancer::maintenance {
+  file { '/usr/share/nginx/www':
+    ensure  => directory,
+    mode    => '0755',
+    owner   => 'deploy',
+    group   => 'deploy',
+    require => Package['nginx'],
```
Comment (deanwilson) [inline modules/loadbalancer/manifests/maintenance.pp]: That package resource actually ensures nginx is absent. Class['nginx::package'] might be a better fit

Comment (alexmuller) [inline modules/loadbalancer/manifests/maintenance.pp]: Oops, thanks for catching that. Fixed.

File: modules/loadbalancer/manifests/maintenance.pp [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# == Class: loadbalancer::maintenance
+#
+# Sets up a machine so that the loadbalancer can be put
+# into maintenance mode.
+#
+class loadbalancer::maintenance {
+  file { '/usr/share/nginx/www':
+    ensure  => directory,
+    mode    => '0755',
+    owner   => 'deploy',
```
Comment (deanwilson) [inline modules/loadbalancer/manifests/maintenance.pp]: Does this need to be owned by deploy to allow fabric pushing of new files?

**Comment by deanwilson** [comment]
Looks good. Merging.

**Pull request merged**